types/meteor/check.d.ts

```diff
@@ -1,24 +1,45 @@
 declare module "meteor/check" {
     module Match {
-        var Any: any;
-        var String: any;
-        var Integer: any;
-        var Boolean: any;
-        var undefined: any;
-        var Object: any;
+        interface Matcher<T> { __impossibleToSatisfy: any; }
```

Comment:
I couldn't figure out a better way to express this - I'm just looking for a type that I can return but nobody else can (otherwise any type trivially satisfies the interface)

Reply:
Aha, "type brand" was the magic phrase I wasn't familiar with. Changed to match more common styles based on some looking around.